athenet/sparsifying/__init__.py

```diff
@@ -1 +1,3 @@
 from sparsify_smallest import *
+from utils import *
+from derest import *
```

Comment:
There was a discussion on this on Slack. More or less, it is DERivatives ESTimation. The problem with "algorithm" name is that it is too much general name for any algorithm.